edi-update-process.Rmd

```diff
@@ -0,0 +1,113 @@
+author: "Badhia Yunes Katz (FlowWest)"
+output: html_document
+---
+
+This guide provides step-by-step instructions to update the Feather River Redd Data EDI package with new data.
+
+## Navigating to the Repository
+
+All data processing and EDI uploads are managed in the same repository.  
+
+1. Navigate to the [project repository](https://github.com/FlowWest/feather-redd/tree/main) and clone it to your local machine.
+
+## Adding Data
+
+1. Before adding the data to the repository, make sure the data contains the following fields:
+
+    * date  
+    * type  
+    * number_redds  
+    * number_salmon  
+    * latitude_m_n  
+    * longitude_m_e  
+    * depth_m  
+    * pot_depth_m  
+    * velocity_m_s  
+    * percent_fines  
+    * percent_small  
+    * percent_med  
+    * percent_large  
+    * percent_boulder  
+    * redd_width_m  
+    * redd_length_m
+
+
+2. Add the Excel file with new data into the `dwr_chinook_redd_survey_data` [folder](https://github.com/FlowWest/feather-redd/tree/main/data-raw/dwr_chinook_redd_survey_data) of your local computer copy, following this naming convention: `yyyy_Chinook_Redd_Survey_Data.xlsx`, replacing "yyyy" with the actual year of the data.
+
+    **Additional Note**: Ensure that the data values are consistent with the expected formats and units. If unsure about the data source or format, consult with the data collection team before proceeding.
+
+## Processing Data
+
+Data processing is handled by one script located in the [data-raw](https://github.com/FlowWest/feather-redd/blob/main/data-raw) folder:  [redd_qc_2014_2023.Rmd](https://github.com/FlowWest/feather-redd/blob/main/data-raw/redd_qc.Rmd)
+
+
+
+1. On the `redd_qc.Rmd` script, add the following code under line 116 to read new data:
+    ```
+    redd_yyyy <- readxl::read_xlsx(here::here("data-raw", "dwr_chinook_redd_survey_data", "yyyy_Chinook_Redd_Survey_Data.xlsx")) |>
+    ```
+    ```
+    clean_names() |>
+    ```
+    ```
+      glimpse()
+    ```
+
+2. After confirming that the data is showing properly, run the `redd_qc.Rmd` script located [here](https://github.com/FlowWest/feather-redd/blob/main/data-raw/redd_qc.Rmd). Note that each year's data was entered differently, so the data cleaning is specific to each year. We will adapt this data cleaning process once the data entry protocol is unified/consistent.
+
+3. The script will generate a CSV called `redd_observations.csv` in the [data folder](https://github.com/FlowWest/feather-redd/tree/main/data).
+
+4. After generating the CSV files, update their metadata (located [here](https://github.com/FlowWest/feather-redd/tree/main/data-raw/metadata)):
+
+    - **Feather Redd** (`feather_redd_metadata.xlsx`): Update the minimum and maximum values for fields such as `date`, `number_redds`, `number_salmon`, etc.
+    - If the time range of the data has changed, update the **project metadata** in `feather_redd_project_metadata.xlsx` (under the Coverage tab).
```

Comment:
Remove "data" before abstract. Can also add methods. So change to "adjust the language in the abstract and methods..."